#ifndef NUEVO_USUARIO_H
#define NUEVO_USUARIO_H

#include <header.h>
#include <mainwindow.h>

/*
namespace Ui {
class nuevo_usuario;
}

class nuevo_usuario : public QDialog
{
    Q_OBJECT

signals:
    void SendData(QString item,QString zona);

public:
    explicit nuevo_usuario(QDialog *parent = 0);
    ~nuevo_usuario();

public slots:
    void ReceivedData_dialog(QString item);

private slots:
    void on_buttonBox_accepted();
    void on_buttonBox_rejected();
    void on_abrirButton_clicked();

private:
    Ui::nuevo_usuario *ui;
};
*/
#endif // NUEVO_USUARIO_H
